src/views/Pools/components/CakeVaultCard/index.tsx

```diff
@@ -35,6 +39,10 @@ const CakeVaultCard: React.FC<{ pool: Pool; account: string }> = ({ pool, accoun
   const stakingTokenPrice = useGetApiPrice(stakingToken.address ? getAddress(stakingToken.address) : '')
   const isLoading = !pool.userData || !userInfo.shares
 
+  if (showStakedOnly && !accountHasSharesStaked) {
+    return <></>
```

Comment:
I think `return null` is more common way to render nothing

Reply:
Changed to null